sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryStatisticsPage.scala

```diff
@@ -114,6 +133,9 @@ private[ui] class StreamingQueryStatisticsPage(parent: StreamingQueryTab)
   def generateStatTable(query: StreamingQueryUIData): Seq[Node] = {
     val batchTimes = withNoProgress(query,
       query.recentProgress.map(p => df.parse(p.timestamp).getTime), Array.empty[Long])
+    val batchToTimestamps = withNoProgress(query,
```

Comment:
Can we create `batchTimes` like `batchToTimestamps.map(_._2)` to avoid having duplicate code?